<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weed_detection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imgs/ral_icon.jpg</normaloff>imgs/ral_icon.jpg</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>¿Desea etiquetar objetos dentro de las imágenes?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="YNC_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>192</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
